src/NuGet.Clients/NuGet.SolutionRestoreManager/VsTargetFrameworks.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace NuGet.SolutionRestoreManager
+{
+    internal class VsTargetFrameworks : VsItemList<IVsTargetFrameworkInfo>, IVsTargetFrameworks
```

Comment:
There are a number of classes whose purpose seems only to be to extract a single member from a type. Can this be done in a more lightweight fashion, e.g. with a helper class? (discussion point only)